app/models/enclosure_url_builder.rb

```diff
@@ -52,6 +52,11 @@ def podcast_episode_expansions(podcast, episode, feed)
     fn = File.basename(base[:path].to_s)
     ex = File.extname(base[:path].to_s)
 
+    # extensions only matter for audio, so default to .mp3 otherwise
+    unless media.try(:medium) == "audio" && %w[.wav .flac].include?(ex)
```

Comment:
It seems like this would change a `.mp4` video to a `.mp3` extension - which is not what we want.  I think we _only_ want to change when medium `== "audio"`.

Can you also add a test to confirm this?  (That it leaves extensions alone for `medium == "anything-else"` ... but alters it for `"audio"`).